src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__img">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="images iMac"
+        />
+      </div>
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__product_code">Product code: 195434</div>
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="reviews">Reviews: 5</div>
+      </div>
+      <div class="price">
+        <div class="text1">Price:</div>
+        <div class="text2">$2199</div>
+      </div>
+      <button class="button">
+        <a
+          data-qa="hover"
+          class="buy"
+          href="#"
+        >
+          BUY
+        </a>
+      </button>
```

Comment:
The button element should not contain the anchor tag. If you need a clickable element that navigates to a new page or resource, use the `<a>` tag with the appropriate 'role' attribute and style it as a button. If you need to submit a form or trigger an action in the page, use the `<button>` tag.